apps/palworld-server-tool/0.9.3/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   pst:
-    image: jokerwho/palworld-server-tool:0.9.3
+    image: jokerwho/palworld-server-tool:0.9.4
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:

```

Comment:
The provided code change appears to be an update from version `0.9.3` of the PalWorld server tool Docker image to version `0.9.4`. There are no significant irregularities or issues with this change, but here is one suggestion for improvement:

### Suggestion:
Consider adding environment variables to control which ports the application listens on within the container. This can enhance flexibility and allow you to deploy the service directly without modification on various hosting environments.

```yaml
services:
  pst:
    image: jokerwho/palworld-server-tool:0.9.4
    container_name: ${CONTAINER_NAME}
    restart: always
    ports:
      - "${PORTS:-81}:81" # Default port if PORTS env variable isn't set
    networks:
```

This way, you don't need to specify a fixed port every time you start up the docker-compose stack; it will fall back to default port `81` unless overridden by setting the `PORTS` environment variable.